apps/mobile/app.json

```diff
@@ -76,6 +77,17 @@
 				"projectId": "9f38f768-16f6-4c47-b354-9745ebda1335"
 			}
 		},
+		"hooks": {
+			"postPublish": [
+				{
+					"file": "sentry-expo/upload-sourcemaps",
+					"config": {
+						"organization": "SENTRY_ORG",
```

Comment:
@desperado1802 will it substitute those vars for SENTRY_ORG and SENTRY_PROJECT here from env vars or we have to hardcode or what? How will it work? In my understanding we should put there correct values so it can upload sourcemaps for stack traces etc to Sentry.io!